app/assets/stylesheets/curated_lists.scss

```diff
@@ -1,5 +1,14 @@
+$light-grey: #f1f1f1;
+$grey: #ccc;
+$dark-grey: #333;
+$white: #fff;
+$light-yellow: #ffff99;
+
 // Hide the new item form.
-html.js {
+.js {
+  // scss-lint:disable SelectorFormat
+  // to fix [W] SelectorFormat: Selector `new_list_item` should be written in hyphenated BEM (Block Element Modifier) format
+  // @TODO: Investigate and remove element if element not used.
   .new_list_item {
```

Comment:
It's in [curated_lists.js](https://github.com/alphagov/collections-publisher/blob/master/app/assets/javascripts/curated_lists.js#L70) but I can't see it anywhere else. It seems like that JS would fail if it can't find it, but I'm not sure if that JS is even running.